mautic/mautic
# [UX] Change bundles statuses from Published to Active

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
In this PR, we introduce a terminology update for bundles statuses to improve user experience. The status label “Published” has been revised to “Active,”. This consistency in terminology aids users in better understanding the status of features, streamlining their workflow. The modification is particularly beneficial for users managing campaigns, segments and webhooks etc, ensuring a cohesive and straightforward interface.

Based on https://community.mautic.org/assemblies/ux-ui-tiger-team/f/59/debates/4

These items will be affected:

- Segments
- Campaigns
- Focus items
- Points (Actions, Triggers)
- Stages
- Users
- Webhooks






#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open something like Webhooks
3. Create a new
4. Look at the label

![image](https://github.com/mautic/mautic/assets/116097999/9c40a046-4df3-42c6-9a0d-699535193bd1)
